website/management/commands/create_csrf_tasks.py

```
@@ -1,323 +1,178 @@
-                
+                <p>Cross-Site Request Forgery (CSRF) is an attack that forces authenticated users to submit unintended requests to a web application.</p>
+
                 <h3>How CSRF Works</h3>
                 <ol>
-                    <li>User logs into a legitimate website (e.g., online banking)</li>
-                    <li>User visits a malicious website while still logged in</li>
-                    <li>Malicious site sends a request to the legitimate site using the user's session</li>
-                    <li>The legitimate site processes the request as if the user intended it</li>
+                    <li>User logs into a legitimate website</li>
+                    <li>User visits a malicious website</li>
+                    <li>Malicious site sends unauthorized request using victim’s session</li>
```

Ruff (0.14.4)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)